### ISSUE TITLE
Disabling PXR_ENABLE_GL_SUPPORT breaks the build

### ISSUE THREAD

**Issue by asluk** [PixarAnimationStudios/USD#972]
### Description of Issue

Build breaks when disabling PXR_ENABLE_GL_SUPPORT

### Steps to Reproduce
python build_scripts\build_usd.py --build-args USD,"-DPXR_ENABLE_GL_SUPPORT=FALSE" --build-monolithic --tests _installdev1

```
pxr\usdImaging\lib\usdAppUtils\frameRecorder.cpp(26): fatal error C1083: Cannot open include file: 'pxr/imaging/glf/glew.h': No such file or directory [pxr\usdImaging\lib\usdAppUtils\usdAppUtils.vcxproj]
```

I have a fix.

**Comment by jilliene** [comment]
Filed as internal issue #USD-5567

**Comment by sunyab** [comment]
Closing old issue, this fix was merged in commit 41278d8 in 19.11.
